docs/docs/reference/changed-features/operators.md

```diff
@@ -113,8 +113,8 @@ The purpose of the `@infix` annotation is to achieve consistency across a code b
       @infix def op[T](x: T)(y: S): R         // ok
       @infix def op[T](x: T, y: S): R         // error: two parameters
 
-      @infix def (x: A) op (y: B): R          // ok
-      @infix def (x: A) op (y1: B, y2: B): R  // error: two parameters
+      @infix def op(this: A)(y: B): R          // ok
+      @infix def op(this A)(y1: B, y2: B): R   // error: two parameters
```

Comment:
```suggestion
      @infix def op(this: A)(y1: B, y2: B): R   // error: two parameters
```

Reply:
Thanks for catching this one @rubenfiszel!